Doc/tutorial/appendix.rst

```diff
@@ -10,6 +10,28 @@ Appendix
 Interactive Mode
 ================
 
+There are two variants of the interactive :term:`REPL`.  The classic
+basic interpreter is supported on all platforms with minimal line
+control capabilities.
+
+On Unix-like systems (e.g. Linux or macOS) with :mod:`curses` and
+:mod:`readline` support, a new interactive shell is used by default.
+This one supports color, multiline editing, history browsing, and
+paste mode.  To disable color, see :ref:`using-on-controlling-color` for
+details.  Function keys provide some additional functionality.
+``F1`` enters the interactive help browser :mod:`pydoc`.
```

Comment:
There's a Sphinx role for keyboard entry:

```suggestion
:kbd:`F1` enters the interactive help browser :mod:`pydoc`.
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-kbd